### ISSUE TITLE
Print on news releases should be a secondary DS button

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#17914]
## Description
When going through the work to upgrade our products to use components, we found an instance of a Print button on the News Release detail pages ([example](https://www.va.gov/boston-health-care/news-releases/trust-increases-among-va-bostons-veteran-patients/)). However, this breaks material honestly because it's a semantic button but looks like a link. 

![Screenshot 2024-04-22 at 2 43 39 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/58dc2549-bda0-4e07-ab3c-551620cb6ddc)

After investigation and discussion, we have determined that this should be updated to be a[ secondary button component](https://design.va.gov/components/button/#default---secondary)

On larger screens (the majority of breakpoints) the print button and Download link can be side-by-side, however can we add an additional 8px of spacing left/right between the 2 elements? (This would make 16px total - 8px by default from the button component and 8 more)

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/0359545d-844f-4b09-b87c-b4f62c09cd05)

Then, on the smallest breakpoint (320), when the elements wrap, we would like to force these elements to be on separate lines with a spacing of 16px top/bottom between these elements:

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/7dcf6740-a53b-4d55-90a1-53025c29f3f2)

## Acceptance Criteria
- [ ] Print is updated to be a secondary button component
- [ ] On larger screen sizes Print and download are side by side with 16px of space between then
- [ ] On the 320 breakpoint, Print and download are stacked with 16px of space between them
- [ ] Requires design review

**Comment by laflannery** [comment]
@aklausmeier Ticket created for the imposter Print button

**Comment by aklausmeier** [comment]
MHV appointments team uses a secondary button for print. More to explore, we would want to review our uses cases where this button may appear next to another button. 

<img width="302" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMStuQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--636500eb5f76439e52d0b0ddbcd109f4dc5c74ac/image.png" alt="image.png" />

**Comment by aklausmeier** [comment]
For immediate fix to accessibility issues, I recommend we update this to a secondary button component without an icon. This keeps in alignment with other use cases on VA.gov.

<img width="93" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMUdyQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--2c24b888ecca963b5227e96271c601c51a183913/Screenshot%202024-05-01%20at%202.10.47%20PM.png" alt="Screenshot 2024-05-01 at 2.10.47 PM.png" />

**Comment by laflannery** [comment]
@aklausmeier One question on this update. On desktop this is how this will look:
![Screenshot 2024-05-29 at 9 05 47 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/ec00c6d4-ebc6-4066-86a8-3b442a5cd849)

But at the 320 breakpoint it wraps:
![Screenshot 2024-05-29 at 9 07 34 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/83ed7b37-c019-450f-8120-70d057eb9eb6)

So my question is, should we force them to 2 lines at this break point:
![Screenshot 2024-05-29 at 9 09 29 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/2ce4197c-d27b-44ba-bdca-f91a6108dd98)

**Comment by aklausmeier** [comment]
@laflannery Let's use 16px spacing between the 2 components, print and download, side by side and when wrapped to next line below. 

At the smaller breakpoint when it begins to wrap to next line, break so full link is on one line below the print button. 

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/137226108/49ef2c83-6eab-491b-ad52-50949af9e98e)

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/137226108/6dccec37-71be-4857-b0f9-8e4f65dcb841)

**Comment by chriskim2311** [comment]
This has been completed as part of the work in this ticket https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17855
